### PR TITLE
Add admin analytics

### LINKED ISSUE #4239: Add admin analytics
Implement admin analytics at `analytics/admin` based on the following :

selection_224

### PR DESCRIPTION
**Pull request by shubham-padia**
fixes #4239

### CODE REVIEW

**Comment by shubham-padia** [comment]
@fossasia/open-event-server-devs Please review

File: app/api/admin_statistics_api/events.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        inflect = dasherize
+
+    id = fields.String()
+    draft = fields.Method("events_draft_count")
+    published = fields.Method("events_published_count")
+    past = fields.Method("events_past_count")
+    total = fields.Method("events_total_count")
+
+    def events_draft_count(self, obj):
+        return Event.query.filter_by(state='draft').count()
```
Comment (niranjan94) [inline app/api/admin_statistics_api/events.py]: Use the `get_count` helper already present in the project. It is faster than this. (https://gist.github.com/hest/8798884)

File: docs/api/api_blueprint.apib [hunk 2]
```diff
@@ -16852,3 +16852,213 @@ Requires Event Co-Organizer Access
...
+    {
+        "data": {
+            "attributes": {
+                "attendee": 0,
+                "verified": 1,
+                "admin": 1,
+                "coorganizer": 0,
+                "super-admin": 1,
+                "organizer": 16,
+                "total": 3,
```
Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Total is not needed right.

File: docs/api/api_blueprint.apib [hunk 3]
```diff
@@ -16852,3 +16852,213 @@ Requires Event Co-Organizer Access
...
+
+    {
+        "data": {
+            "attributes": {
+                "confirmed": 0,
+                "accepted": 1,
+                "submitted": 0,
+                "draft": 0,
+                "rejected": 0,
+                "total": 1,
```
Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Total not needed.

File: docs/api/api_blueprint.apib [hunk 4]
```diff
@@ -16852,3 +16852,213 @@ Requires Event Co-Organizer Access
...
+            Accept: application/vnd.api+json
+
+            Authorization: JWT <Auth Key>
+
++ Response 200 (application/vnd.api+json)
+
+    {
+        "data": {
+            "attributes": {
+                "total": 0,
```
Comment (niranjan94) [inline docs/api/api_blueprint.apib]: Total not needed.

File: app/api/admin_statistics_api/events.py (hunk 1 above)
Comment (shubham-padia) [inline app/api/admin_statistics_api/events.py]: @niranjan94 Resolved

File: docs/api/api_blueprint.apib (hunk 2 above)
Comment (shubham-padia) [inline docs/api/api_blueprint.apib]: @niranjan94 Resolved

File: docs/api/api_blueprint.apib (hunk 3 above)
Comment (shubham-padia) [inline docs/api/api_blueprint.apib]: @niranjan94 Resolved

File: docs/api/api_blueprint.apib (hunk 4 above)
Comment (shubham-padia) [inline docs/api/api_blueprint.apib]: @niranjan94 Resolved

**Comment by shubham-padia** [comment]
@niranjan94 please review

**Pull request closed without merging**